### PR TITLE
Added new capabilities for keyboard enhancements

### PR DESCRIPTION
**Pull request by Ypostolov**
Implements (https://adc.luxoft.com/jira/browse/FORDTCN-7950)

This PR is **not ready** for review.

### Testing Plan
Covered by manual testing plan

### Summary
Current PR adds new capabilities for keyboard enhancements.

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

File: app/model/sdl/Abstract/data.js [hunk 1]
```diff
@@ -1231,6 +1231,30 @@ SDL.SDLModelData = Em.Object.create(
                 "imageSupported": true,
                 "textSupported": true
               }],
+              "keyboardCapabilities":{
```
Comment (AKalinich-Luxoft) [inline app/model/sdl/Abstract/data.js]:
```suggestion
              "keyboardCapabilities": {
```

Comment (AKalinich-Luxoft) [reply app/model/sdl/Abstract/data.js]: Please add extra space after `:` symbol in each line

File: ffw/UIRPC.js [hunk 2]
```diff
@@ -1477,7 +1477,31 @@ FFW.UI = FFW.RPCObserver.create(
                             "longPressAvailable": true,
                             "upDownAvailable": true,
                             "imageSupported": true
-                        }]
+                        }],
```
Comment (AKalinich-Luxoft) [inline ffw/UIRPC.js]: @Ypostolov please revert changes in `UIRPC.js`
Looks like keyboard capabilities are not included for `UI.GetCapabilities` according to API

File: app/model/sdl/Abstract/data.js (hunk 1 above)
Comment (Ypostolov) [reply app/model/sdl/Abstract/data.js]: Fixed in 718d529d03402e216d1b77766a4b819e0a00ebaf
Please take a look.

File: ffw/UIRPC.js (hunk 2 above)
Comment (Ypostolov) [reply ffw/UIRPC.js]: Fixed in 718d529d03402e216d1b77766a4b819e0a00ebaf
Please take a look.

**Review by AKalinich-Luxoft — APPROVED** [review]

**Pull request merged**